fcrepo-export/src/test/java/org/fcrepo/exporter/ExporterTest.java

```diff
@@ -146,6 +152,18 @@ public void testExportContainer() throws Exception {
         Assert.assertTrue(exporter.wroteFile(new File("target/rdf/rest/1.jsonld")));
     }
 
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
I get that (wanting to keep things simple for testing) but remember that Java is a strongly-typed language after all (for some meaning of that ambiguous descriptor) so it's almost never wrong to subtype. :) As to whether to do that here or in some other PR/ticket, that's certainly up to you and the Im/Ex sprinters. I'm not at all trying to block the code as-is, just offering suggestions.